canonical-websites/insights.ubuntu.com
# Updated archive sidebar list with self-generating code

## Done

- Updated archive sidebar list with self-generating code
- Added a custom filter for month abbreviations ‘monthname’

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [a monthly archive page](http://0.0.0.0:8023/archives/2017/01/)

## Notes

* Data goes back further, but not with 12-month coverage
    * Should we be checking for any months that are missing data?
* Obviously, there is no design here, just my guess at something concise

## Issue / Card

Fixes #144

## Screenshots

![image](https://user-images.githubusercontent.com/441217/35593729-9cade082-0608-11e8-95d2-1aa835b116ac.png)
